src/IB/IBRedundantInitializer.cpp

```diff
@@ -1046,7 +1045,7 @@ IBRedundantInitializer::initializeDataOnPatchLevel(const int lag_node_index_idx,
             IntVector<NDIM> periodic_offset;
             for (int d = 0; d < NDIM; ++d)
             {
-                periodic_offset[d] = boost::math::round(periodic_displacement[d] / patch_dx[d]);
+                periodic_offset[d] = int(std::round(periodic_displacement[d] / patch_dx[d]));
```

Comment:
`static_cast<int>`? And, if it needs a cast anyway, should we just omit the call to `std::round` and let the cast do the rounding for us?